homeworks/olya.kosenko_olyabot/homework_6/Task_6_1.js

```diff
@@ -0,0 +1,34 @@
+const user = {};
+// regular first letter in word
+const reg1 = /\b[a-z]/g;
+// regular first + and all numbers in string
+const reg2 = /^\+\d*|\d/g;
+let userName;
+let userPhone;
+Object.defineProperties(user, {
+    fullName: {
+        get() {
+            return userName;
+        },
+        set(value) {
+            userName = value.toLowerCase().replace(reg1, str => str.toUpperCase());
```

Comment:
no mutations in global scope please, better protect the value with function wrapper